test/runtests.jl

```diff
@@ -50,7 +50,9 @@ if anyerrors
     throw("Tests failed")
 end
 
-stdin = joinpath(dirname(@__FILE__), "stdin.sh")
-ENV2 = copy(ENV)
-ENV2["JULIA_HOME"] = JULIA_HOME
-run(setenv(`bash $stdin`, ENV2))
+if !is_windows()
```

Comment:
> Not having the bounds does not give the same result, if you've applied any other bounds for the sake of reproducibility. Products depend on this.

It sounds like a different scenario. Could you give a concrete example where it would matter?

> There's also the parsing of `~` which has changed in 0.6 relative to what every past version of this package expects.

I've wondered about this one as well. My current thinking is that we should just remove all the formula code here. It is now in `StatsModels`. If we do that, it might be possible not to bump the version requirement since `~` wouldn't be used in the package anymore.